Odata-docs/changelog/odatalib-7x.md

```diff
@@ -28,6 +28,77 @@ There was a lot of vestigial code left around from the OData v1-3 days that we
 
 Most of our API improvements fall into the category of namespace simplifications or updating verbiage. The single most impactful change that we made was deciding to merge entity type and complex type in ODataLib. We did this because complex type and entity type are becoming more and more similar in the protocol, but we continue to pay overhead to make things work for both of them.
 
+
+## ODataLib 7.7.0 Release
+
+Following are the changes done after 7.6.4
```

Comment:
Please add the comments the support platforms. It's a breaking change

In old 7.6.4, the package supports .net45 and .netstandard1.1
in the 7.7, please list the support platform for each package.

Spatial:  .net45, .netstandard1.1, netstandard2.0
Edm:  .net45, .netstandard1.1, netstandard2.0
Core:  .net45, .netstandard1.1, netstandard2.0
Client: Spatial:  .net45,  .netstandard2.0